test/HcalDigiPipelineTest.cxx

```diff
@@ -0,0 +1,259 @@
+
+#include "catch.hpp" //for TEST_CASE, REQUIRE, and other Catch2 macros
+
+#include "DetDescr/HcalID.h" //creating unique cell IDs
+#include "Framework/ConfigurePython.h"
+#include "Framework/EventProcessor.h"
+#include "Framework/Process.h"
+
+namespace hcal {
+namespace test {
+
+/**
+ * Energy deposited by one MIP on average
+ * [MeV]
+ */
+static const double MIP_ENERGY = 4.66;
```

Comment:
I used `MIP_ENERGY` in the ECal as the "fundamental" unit because that makes the most sense there.
Maybe your min and max energies would make more sense if you put this in terms of PE?